e2e-tests/gef/cypress/e2e/facility/unissued-facilities/unissued-facilities-bank-review-date.spec.js

```diff
@@ -0,0 +1,159 @@
+          // creates application and inserts facilities and changes status
+          cy.apiCreateApplication(BANK1_MAKER1, token).then(({ body }) => {
+            dealId = body._id;
+            cy.apiUpdateApplication(dealId, token, MOCK_APPLICATION_AIN).then(() => {
+              cy.apiCreateFacility(dealId, CONSTANTS.FACILITY_TYPE.CASH, token).then((facility) => {
+                facilityOneId = facility.body.details._id;
+                cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_ONE);
+              });
+              cy.apiCreateFacility(dealId, CONSTANTS.FACILITY_TYPE.CASH, token).then((facility) =>
+                cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_TWO),
+              );
+              cy.apiCreateFacility(dealId, CONSTANTS.FACILITY_TYPE.CONTINGENT, token).then((facility) =>
+                cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_THREE),
+              );
+              cy.apiCreateFacility(dealId, CONSTANTS.FACILITY_TYPE.CASH, token).then((facility) =>
+                cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_FOUR),
+              );
+              cy.apiCreateFacility(dealId, CONSTANTS.FACILITY_TYPE.CASH, token).then((facility) =>
+                cy.apiUpdateFacility(facility.body.details._id, token, MOCK_FACILITY_FOUR),
+              );
+              cy.apiSetApplicationStatus(dealId, token, CONSTANTS.DEAL_STATUS.UKEF_ACKNOWLEDGED);
+            });
+          });
+        });
+    });
+
+    beforeEach(() => {
+      cy.saveSession();
+      cy.login(BANK1_MAKER1);
+    });
+
+    describe('Visiting from unissued facilities page', () => {
+      beforeEach(() => {
+        cy.visit(relative(`/gef/application-details/${dealId}/unissued-facilities`));
+        unissuedFacilityTable.updateIndividualFacilityButton(0).click();
+
+        aboutFacilityUnissued.issueDateDay().type(dateConstants.threeDaysDay);
+        aboutFacilityUnissued.issueDateMonth().type(dateConstants.threeDaysMonth);
+        aboutFacilityUnissued.issueDateYear().type(dateConstants.threeDaysYear);
+
+        aboutFacilityUnissued.shouldCoverStartOnSubmissionNo().click();
+        aboutFacilityUnissued.coverStartDateDay().type(dateConstants.threeDaysDay);
+        aboutFacilityUnissued.coverStartDateMonth().type(dateConstants.threeDaysMonth);
+        aboutFacilityUnissued.coverStartDateYear().type(dateConstants.threeDaysYear);
+
+        aboutFacilityUnissued.coverEndDateDay().type(dateConstants.threeMonthsOneDayDay);
+        aboutFacilityUnissued.coverEndDateMonth().type(dateConstants.threeMonthsOneDayMonth);
+        aboutFacilityUnissued.coverEndDateYear().type(dateConstants.threeMonthsOneDayYear);
+
+        aboutFacilityUnissued.isUsingFacilityEndDateNo().click();
+
+        aboutFacilityUnissued.continueButton().click();
+      });
+
+      it('should redirect user to the about unissued facility page when clicking back link', () => {
```

Comment:
this could be `should redirect user to ${aboutUnissuedFacilityUrl} when clicking the back link`

Reply:
true, can you put about unissued facility page in " " please